icons/theregister.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>The Register icon</title><path d="M22.368 12.8583c-.846.639-1.719 1.243-2.6119 1.818-.893.5679-1.683.9769-2.378 1.2189-.6949.246-1.1669.199-1.4129-.131-.23-.417-.242-1.028-.028-1.826.21-.802.508-1.628.893-2.4849.381-.782.711-1.58.989-2.39s.413-1.6319.401-2.4579c-.012-.826-.266-1.648-.758-2.4699-.433-.671-1.08-1.175-1.941-1.509-.862-.337-1.822-.504-2.8819-.492-1.06.012-2.112.206-3.1479.58-.808.306-1.551.774-2.23 1.402-.675.627-1.207 1.409-1.5999 2.346-.135.341-.222.794-.27 1.3459-.051.548-.079 1.044-.083 1.489-.004.445-.008.679-.004.699-.433.298-.925.663-1.481 1.096-.556.4289-1.104.9489-1.6399 1.5519-.564.663-1.044 1.326-1.437 1.973-.393.6549-.635 1.1549-.727 1.5049-.048.143-.008.532.119 1.183.127.643.457 1.386 1.001 2.2269.544.841 1.417 1.62 2.616 2.331.012.012.044-.012.091-.067.052-.052.079-.079.083-.087-.02-.012-.167-.171-.429-.476-.262-.302-.516-.699-.758-1.175-.246-.484-.345-.993-.31-1.521.048-.731.512-1.398 1.394-2.005.885-.611 2.3939-1.1429 4.5298-1.5919.607-.155 1.132-.385 1.58-.707.449-.314.687-.492.707-.532-.167 1.703.139 3.26.901 4.653.651 1.1789 1.4929 2.2389 2.5369 3.1758l9.9057-9.8777.013-.052c-.472.369-1.012.79-1.632 1.258zM9.4374 7.3634c-.155.413-.31.826-.465 1.243-.15.417-.234.651-.246.707.028.048.127.163.282.337.163.171.254.286.282.342.012.099-.044.337-.171.707-.123.369-.373.647-.738.8379-.48.179-.929.214-1.346.103-.417-.103-.75-.242-1.008-.413-.2539-.175-.3889-.266-.3969-.286l4.0329-4.3909.016.143c-.008.035-.087.258-.242.67z"/></svg>
```

Comment:
LGTM! Any chance you can get it down to 3 decimals?

Reply:
@adamrusted Done